chain/chain/src/types.rs

```diff
@@ -305,7 +305,7 @@ pub type ReceiptResult = HashMap<ShardId, Vec<ReceiptTransaction>>;
 /// Bridge between the chain and the runtime.
 /// Main function is to update state given transactions.
 /// Additionally handles authorities and block weight computation.
-pub trait RuntimeAdapter {
+pub trait RuntimeAdapter : Send + Sync {
```

Comment:
`Sync` might not be necessary.

Reply:
@nearmax It does not compile without `Sync`.

Reply:
Ok